web/ui/react-app/src/DataTable.tsx

```diff
@@ -59,45 +64,69 @@ class DataTable extends PureComponent<QueryResult> {
 
     let rows: ReactNode[] = [];
     let limited = false;
-    switch(data.resultType) {
+    switch (data.resultType) {
       case 'vector':
-        rows = (this.limitSeries(data.result) as InstantSample[])
-          .map((s: InstantSample, index: number): ReactNode => {
-            return <tr key={index}><td><SeriesName labels={s.metric} format={false}/></td><td>{s.value[1]}</td></tr>;
-          });
+        rows = (this.limitSeries(data.result) as InstantSample[]).map(
+          (s: InstantSample, index: number): ReactNode => {
```

Comment:
Do we want to enforce this `.map()` to happen at the end of the line rather at the beginning of a new one where it's more visible? I don't care much either way, just curious if an explicit rules causes this in particular, or it's the side effect of a more general reformatting rule?

Reply:
I have no idea why it's doing this I've tried to make them look the same, but it keeps doing one one way and the other another 😕 Seems to be a prettier thing again....